web/src/components/AppointmentOnlyIcon.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { styled } from 'styled-components'
+
+import { NoteIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import Icon from './base/Icon'
+import Tooltip from './base/Tooltip'
+
+const Container = styled.div`
+  position: absolute;
+  inset-inline-end: -27px;
+  top: 4px;
+`
+
+const IconContainer = styled.button`
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  width: 18px;
+  height: 18px;
+`
+
+const StyledIcon = styled(Icon)`
+  width: 18px;
+  height: 18px;
+  align-self: center;
+`
+
+const TooltipContent = styled.span`
+  @media ${dimensions.smallViewport} {
+    font-size: 14px;
+  }
+`
+
+const TooltipTitle = styled.div`
+  font-weight: 700;
+  margin-bottom: 8px;
+
+  @media ${dimensions.smallViewport} {
+    font-size: 14px;
+  }
+`
+
+type AppointmentOnlyIconProps = {
+  appointmentUrl: string | null
+}
+
+const AppointmentOnlyIcon = ({ appointmentUrl }: AppointmentOnlyIconProps): ReactElement => {
+  const { t } = useTranslation('pois')
+
+  return (
+    <Container>
+      <Tooltip
+        id='appointment'
+        clickable
+        style={{
+          width: '250px',
```

Comment:
Just checked why this does not work with styled-components. Apparently it is overwritten with `width: max-content` somewhere. Either setting `max-width: 250px` or `width: 250px !important` both allow styling the width with styled-components. Not sure if those are better solutions than this so you might as well keep this. Your decision :)

Reply:
Oh, nice find! Thanks!